src/driver-steelseries.c

```diff
@@ -199,9 +194,8 @@ button_defaults_for_layout(struct ratbag_button *button, int button_count)
 }
 
 static int
-steelseries_get_firmware_version(struct ratbag_device *device)
+steelseries_get_firmware_version(struct ratbag_device *device, int *out_major, int *out_minor)
```

Comment:
For consistency with the codebase: out_major -> major_out ; out_minor -> minor_out